VideoApi/VideoApi.UnitTests/Services/VirtualRoom/GetOrCreateAWitnessVirtualRoomTests.cs

```diff
@@ -70,20 +73,19 @@ public async Task should_create_vmr_with_kinly_if_room_is_not_available()
 
             _mocker.Mock<ICommandHandler>().Setup(x =>
                 x.Handle(It.IsAny<UpdateParticipantRoomConnectionDetailsCommand>())).Callback(() =>
-                expectedRoom.UpdateConnectionDetails(newVmrRoom.Room_label, "ingesturl",
+                expectedRoom.UpdateConnectionDetails(newVmrRoom.Room_label, expectedIngestUrl,
                     newVmrRoom.Uris.Pexip_node,
                     newVmrRoom.Uris.Participant));
 
             _mocker.Mock<IKinlyApiClient>().Setup(x => x.CreateParticipantRoomAsync(_conference.Id.ToString(),
                     It.Is<CreateParticipantRoomParams>(vmrRequest => vmrRequest.Participant_type == "Witness")))
                 .ReturnsAsync(newVmrRoom);
-
             var room = await _service.GetOrCreateAWitnessVirtualRoom(_conference, participant);
-
             room.Should().NotBeNull();
             room.Label.Should().Be(newVmrRoom.Room_label);
             room.PexipNode.Should().Be(newVmrRoom.Uris.Pexip_node);
             room.ParticipantUri.Should().Be(newVmrRoom.Uris.Participant);
+            room.IngestUrl.Equals(expectedIngestUrl).Should().Be(true);
```

Comment:
Being really picky this should probably be

room.IngestUrl.Should().Be(expectedIngestUrl);

To match the format of other tests.